packages/mobile/src/utils/theme.ts

```diff
@@ -57,6 +57,7 @@ export const defaultTheme = {
   staticNeutralLight2: '#AAA7B8',
   staticNeutralLight8: '#F2F2F4',
   staticAccentGreenLight1: '#23AD1A',
+  staticSpecialLightGreen1: '#13C65A',
```

Comment:
nit: In terms of the way our colors are defined, this isn't a static color (i.e. logically considered to never change across themes).
So I think we probably want to omit the `static` prefix here?
There's a broader question about how we expect to map color names in the design system with color names in code. My hope was that they would be as close of an exact match as possible. ('Default/Special/Light Green' -> `colors.specialLightGreen`)

cc @dylanjeffers in case he has any thoughts or suggestions

Reply:
makes sense